### PR TITLE
debian qemu instructions

### PR DESCRIPTION
**Pull request by abe-winter**
add instructions for emulating ARM on a linux laptop

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,12 +122,18 @@ Optionally `-a` can be appended to terminate ALL canon-managed containers.
...
 Docker can be used cross-architecture, such as running arm64 images and toolchains on amd64, and vice versa. This is enabled by default
-on the MacOS versions of docker. For Linux, qemu can be installed and configured. Most distros have a package to do this
-(look for qemu-user-static), or you can run the following to try it out as a one-shot:
+on the MacOS versions of docker.
+
+On x86 Linux, Docker [can use binfmt](https://docs.docker.com/build/building/multi-platform/#qemu) to automatically run ARM containers. On Debian and Ubuntu, you can follow [Debian's qemu instructions](https://wiki.debian.org/QemuUserEmulation) to set this up:
+
+```sh
+sudo apt install binfmt-support qemu-user-static
+```
```
Comment (zaporter-work) [inline README.md]:
```suggestion
On Debian and Ubuntu, you can follow [Debian's qemu instructions](https://wiki.debian.org/QemuUserEmulation) to set this up:

ticktickticksh
sudo apt install binfmt-support qemu-user-static
tickticktick

Most other Linux distributions should have a packaged version of [binfmt](https://github.com/tonistiigi/binfmt) and [qemu-user-static](https://github.com/multiarch/qemu-user-static) that you can install to run containers on different architectures ([docker docs](https://docs.docker.com/build/building/multi-platform/#qemu)).
```

??? Workshopping. No idea if this is better. Just wanted to throw out an alternative.

**Review by zaporter-work — COMMENTED** [review]
You should add @Otterverse because he owns this project.

Seems like a good improvement.

**Comment by abe-winter** [comment]
got it, will tag him when he's back

**Review by Otterverse — APPROVED** [review]
Works for me!

**Pull request merged**